lib/Test/Nginx/Socket.pm

```diff
@@ -3283,6 +3283,20 @@ current test block. Default to "localhost".
 
 Run a piece of Perl code specified as the content of this C<--- init> section before running the tests for the blocks. Note that it is only run once before *all* the repeated requests for this test block.
 
+=head2 post_init
```

Comment:
post_init is not a good name.
I don't know what the init means.

Reply:
IMO, `--- post_init`'s `init` inherited from `--- init`.

Both section can do the same thing, the difference is `--- post_init` is executed later than `--- init`, which allow it to preserve the changes in `t/servroot/conf`, then we can test the stuff within the `conf` folder easier.

A typical scenario is that we are able to test some of the conf files directly in a classic nginx workbench like bellow

```shell
├── conf
│   ├── mime.types
│   └── nginx.conf
└── t
    ├── firmware.pm
    └── sanity.t
```

In this scenario, we can use `--- post_init` to create a symbolic link for `conf/mime.types`, which allow test nginx instance to include it by `include mime.types;`, it do help to reduce the complexity to use `include` in test nginx instance. :)